### PR TITLE
external_script: cast numeric output automatically

### PR DESCRIPTION
**Pull request by lasers**
We cast the output automatically to work with `int`, `float`, `NoneType`, etc better.

EDIT: Addresses https://github.com/ultrabug/py3status/issues/1391.

### CODE REVIEW

**Comment by tobes** [comment]
This implementation is buggy and changes the modules behaviour when not warranted. 

consider this output

`   1   `  <- module crashes
`hello   `  <- changes the output to `hello`
`"hello"` <-changes output to `hello`

I also think `None`, `True`, `False` should remain a string and not be converted

**Comment by tobes** [comment]
ok if you make the variable name consistent `output_text` is a lie and add a comment about why we are doing this they yeah is nice and simple

**Comment by lasers** [comment]
Which PR do we want to use? Yours or mine?  I just saw yours... BIG.

**Comment by tobes** [comment]
I prefer yours with my comments and var name change

File: py3status/modules/external_script.py [hunk 1]
```diff
@@ -76,14 +76,26 @@ def external_script(self):
...
-            output_text = output_lines[0]
+            output = output_lines[0]
             if self.strip_output:
-                output_text = output_text.strip()
+                output = output.strip()
+            # If we get something that looks numeric then we convert it
+            # to a numeric type because this can be helpful. for example:
+            #
+            #   format = "\?if=output>10 File is big!"
+            #   script_path = "cat my_file|wc -l"
```
[the remark this answers is not in the document]
Comment (lasers) [reply py3status/modules/external_script.py]: Sorry. Internet went out. Comment updated. I mention small file too.

**Comment by lasers** [comment]
>output_text is a lie

Renaming it to `output` is a lie too... It should be `first_line`, with `second_line` for color.

This is why I don't want to document `output_full` as I would prefer we deal with all script PRs in one swift PR... which also could switch them over to true `output` along with `line_1`, `line_2`, etc possibly for more unnecessary flexibility and/or line control.

**Comment by lasers** [comment]
Are you sure it's not better if we cast this automatically in `formatter` / `check_valid_condition()`?

Now I wonder about modules that we didn't bother to cast fully or that new (str) things could always could be added in external calls ... or when we change URLs... then we will have to make sure we don't forget about casting them too... on the chance somebody might wanted to compare.

Now I think maybe we need to do this in `getjson` too because of too many questionable URLS.  Next, I don't know where to start with modules since I'm sure what modules the author neglected to partially cast. Module `speedtest` already outputs its JSON correctly, but there are some values that are not likely to be used / compared... but still, we should try and cast them all anyway for the formatter.

**Comment by tobes** [comment]
Cool makes this module super cool again

> Are you sure it's not better if we cast this automatically in formatter

100% bugs should be seen

**Pull request merged**